dbt_coves/tasks/generate/docs.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import nested_scopes
+
+import json
+from pathlib import Path
+from typing import Any, Dict
+
+from rich.console import Console
+
+from dbt_coves.tasks.base import BaseConfiguredTask
+from dbt_coves.tasks.setup.utils import print_row
+from dbt_coves.utils.shell import run_and_capture_cwd
+from dbt_coves.utils.tracking import trackable
+
+console = Console()
+
+
+class DbtCovesGenerateDocsException(Exception):
+    pass
+
+
+class GenerateDocsTask(BaseConfiguredTask):
+    """
+    Task that generates content on local catalog.json
+    """
+
+    @classmethod
+    def register_parser(cls, sub_parsers, base_subparser):
+        subparser = sub_parsers.add_parser(
+            "docs",
+            parents=[base_subparser],
+            help="Merge models from a catalog.json into another one.",
+        )
+        subparser.add_argument(
+            "--merge-deferred",
+            action="store_true",
+            help="Flag to merge deferred models and sources into local catalog",
+            default=False,
+        )
+        subparser.add_argument(
+            "--state",
+            type=str,
+            help="Catalog.json to use as reference for merging",
+        )
+        cls.arg_parser = base_subparser
+        subparser.set_defaults(cls=cls, which="docs")
+        return subparser
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_config_value(self, key):
+        return self.coves_config.integrated["generate"]["docs"][key]
+
+    def _generate_dbt_docs(self):
+        output = run_and_capture_cwd(["dbt", "docs", "generate"], self.config.project_root)
+
+        if output.returncode == 0:
+            deps_status = "[green]SUCCESS :heavy_check_mark:[/green]"
+        else:
+            deps_status = "[red]FAIL :cross_mark:[/red]"
+        print_row(
+            "dbt docs generate",
+            deps_status,
+            new_section=True,
+        )
+        if output.returncode > 0:
+            raise Exception("dbt deps error. Check logs.")
+
+    def _fix_dbt_docs_links(self, docs_path: Path):
+        dbt_docs_index_path = Path(docs_path, "index.html")
```

Comment:
why not just replacing "</head>" by "<base target='_blank'></head>"?

Reply:
Yes, that's better. This method is a copy of Balboa's script, will change